### ISSUE TITLE
Connexion issue

### ISSUE THREAD

**Issue by BenjaminOddou** [Dashlane/dashlane-cli#68]
Hello,

I am trying to connect to the dashlane cli but I have some issues trying to connect:

First I had this when entering my mail:
```shell
? Please enter your OTP code: ERROR: The OTP was already used for this user.
```
Then I revoked accesses of Dashlane CLI sessions via the settings of my account and now when entering my mail:
```shell
? Please enter your OTP code: ERROR: The OTP verification failed. This could be due for example to the user inputing a wrong token.
```

Do you have an idea on how to fix it ? Many thanks in advance 😄

EDIT : 
* I migrated from the dcli-1.0.0 to dcli-1.2.0 / MacOS Ventura 13.2.1 (22D68)  
* I removed all dashlane-cli elements from MacOS keychain

Using any other valid email address returns this message:
```shell
? Please enter the code you received by email: ERROR: Request is malformed
```

**Comment by BenjaminOddou** [comment]
I don't think so, I didn't input an OTP. It's a bit complicated to troubleshoot, tell me if you need me to run some tests.

**Comment by Mikescops** [comment]
I'm a bit confused on how to repro this. You didn't enter any otp?

**Comment by markusstha** [comment]
I have the same problem (mac os, monterey). I just installed the cli and get for the first attempt:

```
? Please enter your OTP code: ERROR: The OTP verification failed. This could be due for example to the user inputing a wrong token.
```

and since then I get

```
? Please enter your OTP code: ERROR: The OTP was already used for this user.
```

I just entered my email, and was not given time to enter any OTP

**Comment by Mikescops** [comment]
I can reproduce on the v1.2.0 from the release but I cannot reproduce on the master branch, I also tried with a packaged version of the master branch and it is fine.

It's my bad I introduced a bug but it was fixed in #64. `v1.3.0` will include the fix.
